packages/cashier/src/stores/account-transfer-store.js

```diff
@@ -336,12 +336,14 @@ export default class AccountTransferStore {
                           platform: account.account_type,
                           is_eu: this.root_store.client.is_eu,
                       })} ${
-                          account.landing_company_short &&
-                          account.landing_company_short !== 'svg' &&
-                          account.landing_company_short !== 'bvi'
-                              ? account.landing_company_short?.charAt(0).toUpperCase() +
-                                account.landing_company_short?.slice(1)
-                              : account.landing_company_short?.toUpperCase()
+                          !this.root_store.client.is_eu
```

Comment:
I think the ternary expressions are considered as warnings in the eslint. (no-nested-ternary)